templates/projects/js/amp_store.js

```diff
@@ -0,0 +1,18 @@
+<script type="text/javascript">
+$(document).ready(function () {
```

Comment:
Minor, but this can be [optimized](https://learn.jquery.com/using-jquery-core/document-ready/) as:

```javascript
$(function() {
  // ...
});
```